decidim-comments/spec/events/decidim/comments/user_mentioned_event_spec.rb

```diff
@@ -41,14 +52,147 @@
         .to include("You have been mentioned in <a href=\"#{resource_path}#comment_#{comment.id}\">#{translated resource.title}</a>")
 
       expect(subject.notification_title)
-        .to include(" by <a href=\"/profiles/#{comment_author.nickname}\">#{comment_author.name} @#{comment_author.nickname}</a>")
+        .to include(" by <a href=\"/profiles/#{author.nickname}\">#{author.name} @#{author.nickname}</a>")
     end
   end
 
   describe "resource_text" do
     it "correctly renders comments with mentions" do
       expect(subject.resource_text).not_to include("gid://")
-      expect(subject.resource_text).to include("@#{comment.author.nickname}")
+      expect(subject.resource_text).to include("@#{author.nickname}")
+    end
+  end
+
+  describe "translated notifications" do
```

Comment:
Can't we refactor this to a shared example? It looks like it's copy-pasting from there and it's like 95% the same.